applications/convection_diffusion_application/convection_diffusion_application_variables.cpp

```diff
@@ -13,17 +13,19 @@
 
 namespace Kratos
 {
-KRATOS_CREATE_VARIABLE(double, MELT_TEMPERATURE_1)
-KRATOS_CREATE_VARIABLE(double, MELT_TEMPERATURE_2)
+
+KRATOS_CREATE_VARIABLE(double, AUX_FLUX)
+KRATOS_CREATE_VARIABLE(double, AUX_TEMPERATURE)
 KRATOS_CREATE_VARIABLE(double, BFECC_ERROR)
 KRATOS_CREATE_VARIABLE(double, BFECC_ERROR_1)
-
+KRATOS_CREATE_VARIABLE(double, DELTA_SCALAR1)
```

Comment:
The names are not very descriptive (why 1 and 2)?

Reply:
These variables were already there before this PR implementations. They can be renamed in another PR if needed.

Reply:
Yes I was just asking